cmd/argocd/commands/app_actions.go

```diff
@@ -67,7 +67,8 @@ func NewApplicationResourceActionsListCommand(clientOpts *argocdclient.ClientOpt
 		ctx := context.Background()
 		resources, err := appIf.ManagedResources(ctx, &applicationpkg.ResourcesQuery{ApplicationName: &appName})
 		errors.CheckError(err)
-		filteredObjects := util.FilterResources(command.Flags().Changed("group"), resources.Items, group, kind, namespace, resourceName, true)
+		filteredObjects, err := util.FilterResources(command.Flags().Changed("group"), resources.Items, group, kind, namespace, resourceName, true)
+		errors.CheckError(err)
```

Comment:
Is this a breaking change in terms of CLI behaviour?

Reply:
I have checked that , only one thing that i would be double check it is about exit code , because i think that log.error return code 1, in case with this function we return code 20 ( that actually also not 0 ) i think it is not breaking change because you have a lot of other places where you exit with code 20